### DISCUSSION TITLE
> Downloading Prisma engines for Node-API for windows [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/2b0c12756921c891fec4f68d9444e18c7d5d4a6a/windows/migration-engine.exe.gz.sha256 failed, reason: connect ETIMEDOUT 18.65.3.19:443

### DISCUSSION THREAD

**Discussion by rajkumar-react1720** [prisma/prisma#13429] [Q&A] [answered]
Hello,

I am getting below error when I try to run npx prisma init

> Downloading Prisma engines for Node-API for windows [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/2b0c12756921c891fec4f68d9444e18c7d5d4a6a/windows/migration-engine.exe.gz.sha256 failed, reason: connect ETIMEDOUT 18.65.3.19:443

prisma version:  "@prisma/client": "^3.13.0",

Please help me out with this error

**Comment by rajkumar-react1720** [comment, marked as answer]
@nikolasburk

**Reply by rajkumar-react1720** [reply]
@pantharshit00  can you please add the missing binary  https://binaries.prisma.sh/all_commits/2b0c12756921c891fec4f68d9444e18c7d5d4a6a/windows/prisma-fmt.exe.gz.sha256 for this time out issue

**Reply by janpio** [reply]
Please stop @ mentioning people directly here in the discussion.

Also the file you link to is available on he server and downloads just fine. This has to be something on your side that is blocking the download. There is not much we can do to help here. Sorry.

**Reply by pantharshit00** [reply]
Yes, please try downloading it via a VPN or some another network

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
